packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -10,7 +10,30 @@ const componentSelector = 'pie-cookie-banner';
 
 export class PieCookieBanner extends LitElement implements CookieBannerProps {
     render () {
-        return html`<h1 data-test-id="pie-cookie-banner">Hello world!</h1>`;
+        return html`
+        <aside data-test-id="pie-cookie-banner" class="c-cookieBanner">
+            <p class="c-cookieBanner-title">Cookies</p>
+            <div class="c-cookieBanner-body">
+                <p>We use our own and third party cookies and other tech to enhance and personalise your user experience,
+                optimize analytics, and show ads with third parties
+                <pie-link variant="inverse">(read our Statement)</pie-link>.
```

Comment:
You'll need to set the `isStandalone` prop to true as well to show the link underline.

Reply:
that's strange, as you can see in the screenshots above the underlines are showing without it. I'll add the prop anyway 👍

Reply:
<img width="702" alt="Screenshot 2023-08-23 at 09 21 33" src="https://github.com/justeattakeaway/pie/assets/16766185/7842438d-febb-4ad3-88bd-9b3d21f49a7b">
@ashleynolan I don't think that prop determines underlines but rather block behaviour